### ISSUE TITLE
Request featured: Page dot navigation.

### ISSUE THREAD

**Issue by Emilybkk** [ThemeFuse/Brizy#334]
There should be option to turn on page dot navigation that jump to blocks. i am not sure where to put in the UI but maybe below the "Page Template" option.

![dot-navigation](https://user-images.githubusercontent.com/13035541/42725182-aa375d14-87a9-11e8-8fd8-92840dda8cdd.png)

**Comment by cmbc58** [comment]
Agree! A feature Divi has (but may not be much used??). I agree this would be an excellent addition. Would be a bit like 
https://designmodo.com/slides/ 
but much cheaper and integrated into WordPress.

**Comment by Emilybkk** [comment]
@cmbc58 Yes. And I would like it to have feature like scroll full-height section too. I don't know how to call it but when you press Spacebar it will directly jump to the next section and it look nice when apply to full-height section.

**Comment by Emilybkk** [comment]
@cmbc58 Exactly like the the link in that website you mention. Like this http://a-say.co/#contact

**Comment by cmbc58** [comment]
Do you know about this plugin. I've only just found it and not tried it out yet, but it looks promising and the Pro version is not expensive...
https://en-gb.wordpress.org/plugins/wp-floating-menu/

**Comment by Emilybkk** [comment]
No never heard of it. I not a fan of vertical menu haha
